cider-velox/src/substrait/VeloxPlanFragmentToSubstraitPlan.cpp

```diff
@@ -109,13 +108,47 @@ void VeloxPlanFragmentToSubstraitPlan::reconstructVeloxPlan(
       planBuilder_->addNode([&](std::string id, core::PlanNodePtr input) {
         return std::make_shared<ValuesNode>(valuesNode->id(), valuesNode->values());
       });
+    } else if (auto joinNode =
+                   std::dynamic_pointer_cast<const AbstractJoinNode>(*riter)) {
+      const auto& joinLeftSource = joinNode->sources()[0];
+      const auto& joinRightSource = joinNode->sources()[1];
+
+      auto leftValuesNode = std::make_shared<ValuesNode>(
```

Comment:
What about the case if there not only have one join node ,such as , it's filter->project->join?

Reply:
not support now , consider it as TODO item